app/serializers/api/v1/user_serializer.rb

```diff
@@ -7,7 +7,7 @@ class UserSerializer < ApplicationSerializer
     has_one :image, serializer: ImageSerializer
     has_one :address, serializer: AddressSerializer
     has_many :user_roles, serializer: UserRoleSerializer
-    # has_many :organisations_users, serializer: OrganisationsUserSerializer
+    has_many :organisations_users, serializer: OrganisationsUserSerializer
```

Comment:
I've made a comment in `OffersController` to ensure this serializer is only included if user is staff